docs/data/base/components/modal/ServerModal.js

```diff
@@ -55,48 +55,54 @@ const grey = {
   900: '#1C2025',
 };
 
-const blue = {
-  200: '#99CCF3',
-  300: '#66B2FF',
-  700: '#0059B2',
-};
-
 const Modal = styled(BaseModal)`
   position: fixed;
   z-index: 1300;
   inset: 0;
   display: flex;
   align-items: center;
   justify-content: center;
-  padding: 8px;
-  background: inherit;
-  font-family: 'IBM Plex Sans', sans-serif;
+
+  &.MuiModal-hidden {
+    visibility: hidden;
+  }
 `;
 
 const style = {
-  position: 'relative',
   width: 400,
-  height: 120,
 };
 
 const ModalContent = styled(Box)(
   ({ theme }) => `
-  font-family: IBM Plex Sans, sans-serif;
-  font-weight: 600;
-  font-size: 0.875rem;
-  line-height: 1.5;
-  padding: 8px 16px;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  overflow: hidden;
+  background-color: ${theme.palette.mode === 'dark' ? grey[900] : '#FFF'};
   border-radius: 8px;
-  color: white;
-  transition: all 150ms ease;
-  background: ${theme.palette.mode === 'dark' ? grey[900] : '#fff'};
   border: 1px solid ${theme.palette.mode === 'dark' ? grey[700] : grey[200]};
-  color: ${theme.palette.mode === 'dark' ? grey[200] : grey[900]};
-  box-shadow: 0 1px 2px 0 rgb(0 0 0 / 0.05);
+  box-shadow: 0px 4px 12px ${
+    theme.palette.mode === 'dark' ? 'rgba(0,0,0, 0.50)' : 'rgba(0,0,0, 0.05)'
```

Comment:
```suggestion
    theme.palette.mode === 'dark' ? 'rgba(0,0,0, 0.5)' : 'rgba(0,0,0, 0.05)'
```

Nit: we don't need that 0 😬